packages/render-fidelity-tools/test/results-viewer.html

```diff
@@ -26,13 +26,79 @@
   <!-- Web Components polyfill is required to support Edge and Firefox < 63: -->
   <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
 <style>
-body {
-  margin: 1em;
-  background-color: #f7f7f7;
-}
+  @import url('https://fonts.googleapis.com/css?family=Rubik:400,500');
+  body {
+    margin: 1em;
+    background-color: #f7f7f7;
+    font-family: 'Rubik', sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+    color: rgba(0,0,0,.87);
+    font-weight: 400;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  p {
+    max-width: 1000px;
+    text-align: left;
+  }
 </style>
 </head>
 <body>
+  <div align="center">
+    <h2>Render Fidelity Comparison Results</h1>
+  
+    <p>The purpose of <a href="https://www.khronos.org/gltf/"
+    target="_blank">glTF</a> is to standardize Physically-Based Rendering (PBR)
+    materials such that you can be confident your model will appear as intended
+    in any lighting environment in any renderer. This is a very ambitious goal,
+    as real-time rendering at this level of quality is still very much an area
+    of active research with improvements being made constantly. This site
+    demonstrates where we are on that path to convergence and highlights areas
+    that could still use improvement. We are comparing the most popular web
+    renderers: <a href="https://threejs.org/" target="_blank">three.js</a>
+    (represented by &lt;model-viewer&gt;), <a
+    href="https://google.github.io/filament/webgl/"
+    target="_blank">filament.js</a>, and <a href="https://www.babylonjs.com/"
+    target="_blank">babylon.js</a>. If any other renderers would like to be
+    included, please open a PR adding them to the <a
+    href="https://github.com/google/model-viewer/tree/master/packages/render-fidelity-tools/src/components/renderers"
+    target="_blank">render-fidelity-tools</a>
+    package.</p>
+
+    <p>In addition, we are showing a ground-truth render provided by <a
```

Comment:
maybe renderer instead of render? but im bad in in English , if they mean the same thing then just ignore it .

Reply:
In this case it is a render (image) provided by a renderer (dspbr-pt).